charts/loki-simple-scalable/values.yaml

```diff
@@ -142,8 +142,8 @@ loki:
       requestTimeout: "0s"
       enableHttp2: true
     local:
-      chunks_directory: /var/loki/chunks
-      rules_directory: /var/loki/rules
+      chunksDirectory: /var/loki/chunks
```

Comment:
this is very nit-picky, so I apologize, but I'd rather keep the key here `chunks_directory` and `rules_directory` as that more closely matches what you would pass to loki itself. I would prefer fixing this by fixing the helper function instead, WDYT?

Reply:
I chose this way around because if you look at the `gcs` or `s3` configs right above it, they use camel case. Also in the entire `values.yaml` the only other example of a snake case key I found is `loki.commonConfig.path_prefix`. That being said, I leave the decision up to you - just tell me if I should change this :)